Pano/Views/LoginRegister/Buttons/withAppleButton.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  withAppleButton.swift
+//  Pano
+//
+//  Created by Serdar Onur KARADAĞ on 22.07.2023.
+//
+
+import SwiftUI
+
+struct withAppleButton: View {
```

Comment:
We can never start a class name with a lowercase letter. I would suggest using something like `SignInWithAppleButton`.